contracts/factory/src/tests/config.rs

```diff
@@ -308,35 +311,14 @@ fn doesn_not_change_whitelisted_accounts_when_removing_non_existent() {
 
     let to_remove = vec![&env, Address::generate(&env)];
 
-    factory.update_whitelisted_accounts(&admin.clone(), &vec![&env], &to_remove);
+    factory.update_config(&None, &None, &None, &None, &None, &Some(to_remove), &None);
 
     let config = factory.get_config();
 
     assert!(config.whitelisted_accounts.contains(admin));
     assert!(config.whitelisted_accounts.len() == 1);
 }
 
-#[should_panic(expected = "Factory: Update whitelisted accounts: You are not authorized!")]
-#[test]
-fn fails_to_update_whitelisted_accounts_when_not_authorized() {
-    let env = Env::default();
-    env.mock_all_auths();
-    env.cost_estimate().budget().reset_unlimited();
-
-    let admin = Address::generate(&env);
-    let first_wl_addr = Address::generate(&env);
-    let second_wl_addr = Address::generate(&env);
-
-    let factory = deploy_factory_contract(&env, admin.clone());
-
-    let to_add = vec![&env, first_wl_addr.clone(), second_wl_addr.clone()];
-    factory.update_whitelisted_accounts(&admin.clone(), &to_add, &vec![&env]);
-
-    let to_remove = vec![&env, admin.clone()];
-
-    factory.update_whitelisted_accounts(&Address::generate(&env), &vec![&env], &to_remove);
-}
```

Comment:
Why was this one removed?

Reply:
two reasons - now we don't have `fn update_whitelisted_accounts()` - we move this code into `fn update_config()`; to require authentication, I simply ask the admin to authorize and skip the sender input argument in general. I think it's redundant to have that and then check if `sender ==admin`.

so in short - test is obsolete